internal/provider/provider.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 var (
-	providerVersion = "dev"
+	version = "dev"
```

Comment:
this has been made to follow the ideology of Go, which forbids to prefix variables with package name